### DISCUSSION TITLE
Will adding more source images improve resemblance?

### DISCUSSION THREAD

**Discussion by MW0973** [hacksider/Deep-Live-Cam#742] [General]
Its a question, coming from an amateur. Is adding more pictures of a source face a feature to even consider in the future? Could it improve faithfulness to the source identity?
Genuinely curious if its something that is possible with underlying inswapper128 model, and if not, why

**Comment by KRSHH** [comment]
I don't think more source images from different angles would make it any more more accurate. It comes down to resemblance between target and source face, like skin color and face structure/shape etc. Although there is room for improvement in accuracy of expressions and resolution of the model for higher end PCs.

**Comment by hacksider** [comment]
This is a pretrained model already and adding more image wont do any good. Unless insightface release their 256 and 512 model, then this is the best we can do as of now until we find another alternative.
